activation/post.go

```diff
@@ -371,7 +392,15 @@ func (mgr *PostSetupManager) findCommitmentAtx(ctx context.Context) (types.ATXID
 		mgr.logger.Info("ATXs synced - selecting commitment ATX")
 	}
 
-	atx, err := atxs.GetIDWithMaxHeight(mgr.db, types.EmptyNodeID)
+	atx, err := findFullyValidHighTickAtx(
+		context.Background(),
+		mgr.db,
+		types.EmptyNodeID,
+		mgr.goldenATXID,
+		mgr.validator,
+		mgr.logger,
+		assumeValidBefore(time.Now().Add(-mgr.postValidityDelay)),
+	)
```

Comment:
What other component would do it? I would need to create something like:
```go
type ValidChainGetter interface {
  GetAtxWithValidChain(...) (types.ATXID, error)
```

Is it worth it though? Maybe you have some ideas?

Reply:
I don't want to put too much effort into this because we will eventually remove initialization from the node. A lot of smeshers already use `postcli` to more efficiently use their available HW resources and I believe Smapp can be adjusted to also initialize using `postcli` before starting the node (needs to be checked with the Smapp team first).